### ISSUE TITLE
Enhance tab component keyboard and screenreader guidance

### ISSUE THREAD

**Issue by anandamaryon1** [nhsuk/nhsuk-service-manual#1981]
Suggestion from a recent Accessibility Audit.

Add content to explain how to operate tabs via keyboard, and for how a screenreader should announce it. 

From Dig Inclusion: "User should be able to change tab via Tab key and Arrow keys, but should not move tab unless pressing Enter key. Screenreader should announce the current tab and the total number of tabs (similar to lists, e.g. 2 of 4)."

**Comment by anandamaryon1** [comment]
Updates live
